littleballoffur/backend.py

```diff
@@ -1,9 +1,9 @@
 import random
-import numpy as np
-import networkx as nx
-import networkit as nk
 from typing import List, Tuple
 
+import networkit as nk
```

Comment:
@benedekrozemberczki this is from isort, which I want to adopt. It keeps things sorted in a PEP compliant way.